client/authority-discovery/src/worker/addr_cache.rs

```diff
@@ -200,14 +201,13 @@ mod tests {
 	impl Arbitrary for TestMultiaddr {
 		fn arbitrary(g: &mut Gen) -> Self {
 			let seed = (0..32).map(|_| u8::arbitrary(g)).collect::<Vec<_>>();
-			let peer_id = PeerId::from_multihash(
-				Multihash::wrap(multihash::Code::Sha2_256.into(), &seed).unwrap(),
-			)
-			.unwrap();
+			let peer_id =
+				PeerId::from_multihash(Multihash::wrap(Code::Sha2_256.into(), &seed).unwrap())
+					.unwrap();
```

Comment:
In case you don't want the explicit `multihash_codetable` dependency here, you might as well generate the `PeerId` via a `ed25519` key. For example:

``` rust
    let mut bytes = [0u8; 32];
    // Fill `bytes` via `seed`.

    identity::Keypair::ed25519_from_bytes(bytes).expect("only errors on wrong length").public().peer_id();
```

Reply:
I think it's fine given we are using it in multiple places, not just here